app/models/invoice_item.rb

```diff
@@ -0,0 +1,13 @@
+class InvoiceItem < ApplicationRecord
+  before_save :price_to_dollars
```

Comment:
This is something I've never done before but is an amazingly easy way to perform some action on data before it is added to database